matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/RoomSummaryQueryParams.kt

```diff
@@ -94,6 +100,7 @@ data class RoomSummaryQueryParams(
      * [roomSummaryQueryParams] and [spaceSummaryQueryParams] can also be used to build an instance of [RoomSummaryQueryParams].
      */
     class Builder {
+        var roomId: QueryStringValue = QueryStringValue.NotContains(RoomLocalEcho.PREFIX)
```

Comment:
I am wondering if we could have a roomId which contains `!local.`, but after checking [in the doc](https://spec.matrix.org/v1.3/appendices/#identifier-grammar), this is not possible, so we are good.